glossary.yml

```diff
@@ -358,6 +375,23 @@
       A unit of information representing representing alternatives, yes/no,
       [true](#true)/[false](#false). In computing a state of either 0 or 1.
 
+- slug: blob
+  en:
+    term: "binary large object"
+    acronym: "BLOB"
+    def: >
+      Data that is stored in a database without being interpreted in any way,
+      such as an audio file. The term is also now used to refer to data
+      transferred over a network as uninterpreted bits.
```

Comment:
Should there be a mention of this usage in git since it's common to see this term in a github URL?

Reply:
Added - thanks.